resources/EPvalidator.rb

```diff
@@ -244,7 +244,7 @@ def self.run_validator(hpxml_doc)
 
       ## [HeatingType=Furnace]
       "/HPXML/Building/BuildingDetails/Systems/HVAC/HVACPlant/HeatingSystem[HeatingSystemType/Furnace]" => {
-        "../../HVACDistribution[DistributionSystemType/AirDistribution | DistributionSystemType[Other='DSE']]" => one_or_more, # See [HVACDistribution]
```

Comment:
I guess these changes would make those files (mostly reference home hpxmls) with DSE approach not passing EPvalidator check.